### DISCUSSION TITLE
Allow file / file(s) interfaces to have default values on upload

### DISCUSSION THREAD

**Discussion by philleepflorence** [directus/directus#3116] [Feature Requests]
Some advanced interfaces that have one or more fields can also have default values.
### Example
**File Interface**
Folder, tags, et al can be the same for several collections (so the user would not need to add over and over).
Using a **JSON** or **Repeater** interface, the admin users can add as many defaults.
JSON seems easier in case we need to add more properties to each object in the array.
```
[
  {
  	"field": "<field>",
	"value": "<value>",
	"condition": "<condition>"
  }
]
```

**Comment by rijkvanzanten** [comment]
Example use case:

Set `directus_files.tags` to `article, featured_image` when uploading a file to the `articles.featured_image` field

**Reply by philleepflorence** [reply]
YES!!!!

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
